spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from collections.abc import Mapping
+
+from cerberus.validator import (
+    Validator as cerberus_validator,
+    DocumentError,
+    SchemaError,
+)
+from spidermon.contrib.validation.validator import Validator
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator, use_default_translator=use_default_translator
+        )
+        if schema is not None:
```

Comment:
Do let me know if this is something that would be good to have.

Reply:
This is fine.